iree/compiler/Dialect/HAL/Transforms/TranslateExecutables.cpp

```diff
@@ -53,19 +63,14 @@ class TranslateExecutablesPass
     auto targetOps = llvm::to_vector<4>(
```

Comment:
can you see if it works to change this to a pass on ExecutableTargetOp (nested in ExecutableOps) instead of ExecutableOps and then doing the loop here? we couldn't do that before IIRC but conceptually should be able to (the backend pipelines shouldn't be mutating the parent ExecutableOp)

Reply:
Done.